### PR TITLE
"upgrading" guide: Rails 5.0 gotcha with disable_with

### PR DESCRIPTION
**Pull request by henrik**
We caught this during upgrading because we also read the release notes and figured out it could be an issue, but seems a good idea to flag it here.

Assuming we're not the last of the stragglers :)

### CODE REVIEW

File: guides/source/upgrading_ruby_on_rails.md [hunk 1]
```diff
@@ -967,6 +967,14 @@ interacting with models, but also affects `:default` column settings in `db/sche
 It is recommended that you do not set columns equal to a `String`, but pass a `Hash`
 instead, which will be converted to and from a JSON string automatically.
 
+### Submit Buttons `disable_with` by Default
+
+In Rails 5.0, any `submit_tag` or `f.submit` is automatically disabled on submit to
+prevent double submits. If you have any client-side validation, this can be an issue.
+Opt out on a button-by-button basis with e.g.
+
+    <%= submit_tag "Submit", data: { disable_with: false }} %>
```
Comment (olleolleolle) [inline guides/source/upgrading_ruby_on_rails.md:976]:
```suggestion
    <%= submit_tag "Submit", data: { disable_with: false } } %>
```

Could balance the individual `}` sign. If you wanted to!

Comment (olleolleolle) [reply guides/source/upgrading_ruby_on_rails.md:976]: I took a double peek and saw that the code had indeed an unbalanced number of `{` and `}` .

**Review by olleolleolle — APPROVED** [review]
👍 LGTM! This was useful!

**Comment by vipulnsward** [comment]
Not sure if we should add to this guide per-se. 
This option has existed before 5.x and also gets a mention on https://github.com/rails/rails/blob/5-0-stable/actionview/CHANGELOG.md  + configuring guide: `config.action_view.automatically_disable_submit_tag`

**Comment by henrik** [comment]
Yeah, I spotted the option in the release notes – my thinking was that this change has a fairly high likelihood of breaking apps, if there's any client-side validation, so it's worth mentioning as a gotcha in the upgrade guide itself. Figured one more item on that list would help more than it hurt.

But I can also see the reasoning in not wanting that list getting too long. Do with this what you will :)

**Comment by vipulnsward** [comment]
Thanks for the contribution, but yeah, we should keep it limited to highlight major changes.

**Comment by henrik** [comment]
Sure. Thanks for looking!

**Pull request closed without merging**